modules/status-roles/events/presenceUpdate.js

```diff
@@ -0,0 +1,22 @@
+const {localize} = require("../../../src/functions/localize");
+
+module.exports.run = async function (client, newPresence) {
+
+    if (!client.botReadyAt) return;
+    const moduleConfig = client.configurations['status-roles']['config'];
+    const roles = moduleConfig.roles;
+    const status = moduleConfig.words;
+    const member = newPresence.member;
+
+    if (newPresence.activities.length > 0) {
+        if (status.some(word => newPresence.activities[0].state.includes(word))) {
```

Comment:
- `newPresence.activities[0].state` can be null, as definied [here](https://discord.js.org/#/docs/discord.js/stable/class/Activity?scrollTo=state)
- This is case sensitive, but that's not ideal: Try lower (or upper)-casing both values